wal/wal.go

```diff
@@ -448,16 +448,17 @@ func (w *WAL) cut() error {
 		return err
 	}
 
-	if err = os.Rename(newTail.Name(), fpath); err != nil {
+	if err = os.Rename(tempTail.Name(), fpath); err != nil {
 		return err
 	}
 	if err = fileutil.Fsync(w.dirFile); err != nil {
 		return err
 	}
 
-	newTail.Close()
+	tempTail.Close()
```

Comment:
instead of having `tempTail`, a here comment might be better for code clarity:
```go
// reopen newTail with its new path so calls to Name() match the wal filename format
```